frontend/src/features/properties/components/forms/ParcelDetailForm.scss

```diff
@@ -39,6 +39,16 @@
         height: 2rem;
       }
     }
+    .page-item.error.active a {
```

Comment:
Display a solid red box on the active page if it is in error. Display a red border otherwise